bin/blanksky_sample

```diff
@@ -100,67 +100,74 @@ def get_par(argv):
 
     ## error out if there are spaces in absolute paths of various parameters
     if " " in os.path.abspath(params["infile"]):
```

Comment:
Not for this PR, but it looks like the "check for spaces" check could be made into a routine, something like

```
def check_no_spaces(infile, filetype):
    abspath = os.path.abspath(infile)
    if " " not in abspath:
        return

    return OSError(f"The asolute path for the {filetype} file, {abspath}, cannot contain any spaces")
```

and then called with something like

```
    for inkey, filetype in [("infile", "source evemts"), ("bkgfile", "blanksky background"), ...]:
        check_no_spaces(params[inkey], filetype)
```

[although I haven't gone through to see if this handles all cases]